mne/report.py

```diff
@@ -2058,3 +2062,60 @@ def _fix_global_ids(html):
         html = re.sub('id="###"', 'id="%s"' % global_id, html, count=1)
         global_id += 1
     return html
+
+
+###############################################################################
+# Scraper for sphinx-gallery
+
+_SCRAPER_TEXT = '''
+.. only:: builder_html
+
+    .. container:: row
+
+        .. rubric:: The `HTML document <{0}>`__ written by :meth:`mne.Report.save`:
+
+        .. raw:: html
+
+            <iframe class="sg_report" sandbox="allow-scripts" src="{0}"></iframe>
```

Comment:
what if you just put the entire report string here? ... just a thought.

Reply:
If you mean ..raw:html directly, the report has html header information and tags so it should work worse. If you mean srcdoc in the inline tag I tried and it was worse than a separate file

Reply:
okay let's keep it separate